src/main/java/com/jensjansson/ce/views/persons/EditorView.java

```diff
@@ -249,6 +250,14 @@ protected void editPerson(Person person) {
         list.setTopic(topicId);
         if (topicId != null) {
             BotRunner.onUserJoined(topicId, localUser, person, personService);
+            CollaborationEngine.getInstance().openTopicConnection(this, topicId,
```

Comment:
We should also close the topic connection at some point with the `Registration` returned from `openTopicConnection`. Now we're opening a new connection every time when selecting a new item, but never closing them.

The callback can be asynchronous, so we can't just remove the connection right after opening, or it's possible that the callback is never called.

I suggest that we do it the same way as the component implementations: Store the `Registration` as a field and when changing the topic (selecting a new item), removing the previous registration if any.

Reply:
I added something that I think corresponds with what you said. Does it look good?

Reply:
Yep! :+1: